app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+<div class='sign-in'>
+  <div class="field">
+    <%= f.text_field :name, placeholder: 'Name' %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: 'Password' %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+   <br>
+  <span> Already have an account ? </span> <%= link_to "Sign In", new_user_session_path %>
+
+ 
+  </div>
```

Comment:
- [ ] [optional] @yosefHesham it would have been nice if your login and registration page followed the wireframe design, check the comparison below

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/224127392-5230941a-2de5-4f52-b54f-41f3b80d4cd7.png)
| ![image](https://user-images.githubusercontent.com/61408860/224127494-410d4c4d-5080-43b8-9410-9187487cd935.png)